README.md

```diff
@@ -836,6 +836,12 @@ ha_cluster_constraints_ticket:
 You may take a look at
 [an example](#creating-a-cluster-with-resource-constraints).
 
+#### `ha_cluster_manage_firewall`
+
+boolean, default: `false`
+
+Manage the `firewall high-availability service` as well as the `fence-virt port`.
```

Comment:
Now that firewall is not configured by default, README must also specify what users do when they want to manage firewall manually. I.e. enable the high-availability service and the 1229/tcp port.

Reply:
If firewall was previously configured by default, then we should make the default value for `ha_cluster_manage_firewall` to be `true`

Reply:
Right. That's what I was worried. Otherwise `ha_cluster_manage_firewall: false` would wipe it out... I'm adding the code.

Reply:
Please take a look at https://github.com/linux-system-roles/ha_cluster/pull/63/commits/0ae86fb860df3ff09717d38e34e244deabad1080.

Reply:
I would prefer to have `ha_cluster_manage_firewall` default to `true`
and just have
```
if ha_cluster_manage_firewall is true:
    enable firewall ports
else:
    disable firewall ports
```
I'm not sure what is the problem that we're trying to solve by adding the more complex code.

Reply:
If the default value is `true`, it always configures and enables the firewall, which is different from the current behavior. Is it ok?

Reply:
> If the default value is `true`, it always configures and enables the firewall, which is different from the current behavior. Is it ok?

I think so - I think that's what the author originally intended (@tomjelinek ?), but we didn't have a firewall role at that time.

Reply:
Assuming it's ok to always configure firewall, I pushed a new commit. Thanks!

Reply:
Thank you for your input, @tomjelinek. Glad to hear it does not break our customers' needs.